lambdas/tests/unit/utils/test_utilities.py

```diff
@@ -34,6 +37,28 @@ def test_decapitalise_keys():
     assert actual == expected
 
 
+def test_get_pds_service_returns_stubbed_pds_when_true(monkeypatch):
+    monkeypatch.setenv("PDS_FHIR_IS_STUBBED", "True")
+
+    response = get_pds_service()
+
+    assert isinstance(response, MockPdsApiService)
+
+
+def test_get_pds_service_returns_stubbed_pds_when_unset():
+    response = get_pds_service()
+
+    assert isinstance(response, MockPdsApiService)
+
+
+def test_get_pds_service_returns_real_pds(monkeypatch):
+    monkeypatch.setenv("PDS_FHIR_IS_STUBBED", "False")
+
+    response = get_pds_service()
+
+    assert isinstance(response, PdsApiService)
+
+
```

Comment:
I've noticed that in `lambdas/utils/utilities.py`, `get_pds_service()` checks `PDS_FHIR_IS_STUBBED` against both "False" and "false" values.
Could we modify these tests to also take into account if the boolean value starts with a lowercase? As testing may require us to modify the `PDS_FHIR_IS_STUBBED` value manually on the AWS console, I could see ourselves interchangeably setting uppercase or lowercase values.

Reply:
Sure, will make these parameterized.

Reply:
This is a good point actually, the terraform `contains()` which determines this value will set to a lower case `true/false` our primary check should be for a lower case, but having both would be airtight